tests/main/interfaces-mount-control/task.yaml

```diff
@@ -0,0 +1,73 @@
+summary: Test for the mount-control interface
+
+environment:
+    MOUNT_SRC: /var/tmp/test-snapd-mount-control
+    SNAP_COMMON: /var/snap/test-snapd-mount-control/common
+    SNAP_NAME: test-snapd-mount-control
+
+prepare: |
+    mkdir -p "$MOUNT_SRC/dir1"
+    echo "Something" > "$MOUNT_SRC/file1"
+
+restore: |
+    rm connect_error.log
+    rm -rf "$MOUNT_SRC"
+
+execute: |
+    MOUNT_DEST="$SNAP_COMMON/target"
```

Comment:
does it not work to define this in the environment section?

Reply:
It does :-) Updated.